dll/win32/shell32/folders/CDrivesFolder.cpp

```diff
@@ -345,7 +345,7 @@ HRESULT CALLBACK DrivesContextMenuCallback(IShellFolder *psf,
         {
             // pdtobj should be valid at this point!
             ATLASSERT(pdtobj);
-            hr = SH_ShowDriveProperties(wszBuf, pdtobj);
+            hr = SH_ShowDriveProperties(wszBuf, pdtobj) ? 0 : (HRESULT)-1;
```

Comment:
```suggestion
            hr = SH_ShowDriveProperties(wszBuf, pdtobj) ? `S_OK` : `E_UNEXPECTED`;
```
Alternatively, make `SH_ShowDriveProperties` return an `HRESULT`